code/modules/mining/remote_fob/remote_fob_fobdrone.dm

```diff
@@ -17,6 +17,7 @@ GLOBAL_LIST_INIT(blocked_remotebuild_objs, typecacheof(list(/obj/machinery/compu
 /mob/camera/aiEye/remote/fobdrone/Initialize()
 	. = ..()
 	starting_area = get_area(loc)
+	set_light(3)
```

Comment:
use set_light_on, set_light is meant for static lights. That's just a performance issue

Reply:
Sure. Will do. I'll make it a little more potent too as 3 is actually like PDA tier.